frontend/src/scenes/session-recordings/player/inspector/components/PlayerInspectorListItem.tsx

```diff
@@ -35,7 +35,7 @@ const typeToIconAndDescription = {
         tooltip: 'Console log',
     },
     [SessionRecordingPlayerTab.NETWORK]: {
-        Icon: IconGauge,
+        Icon: IconDashboard,
```

Comment:
renaming stands out more here...

Reply:
@corywatilo what is our thinking on the names of icons going forward? Even migrating between the two systems I found it quite difficult to know what icons we had and what they might be called.

Do we want to allow any kind of aliasing of names? Maybe something in the PostHog icons repo would help. Open to working on this if you have a sense of what might work

Reply:
I've been going by literal names of what the icons represent (except for a handful). Easy enough to reference from the screenshot at https://github.com/posthog/icons for the time being.